### PR TITLE
Merge develop release to master.

### PR DESCRIPTION
**Pull request by dnduffy**
Stable release 1.0.0.

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="ASPNET1" value="https://dotnet.myget.org/F/aspnet1/api/v3/index.json" />
```
Comment (kartang) [inline NuGet.config]: We might not need this feed anymore, especially in master. Please remove this.

Comment (dnduffy) [inline NuGet.config]: Oops yes, thanks for catching that.

File: test/WebApiShimFw46.FunctionalTests/project.json [hunk 2]
```diff
@@ -12,33 +12,33 @@
...
     "netcoreapp1.0": {
       "imports": [
         "dnxcore50",
-        "dotnet5.4",
+        "dotnet5.6",
         "portable-net451+win8"
       ],
       "dependencies": {
-        "Microsoft.Extensions.CodeGenerators.Mvc": "1.0.0-*",
+        "Microsoft.Extensions.CodeGenerators.Mvc": "1.0.0-rc1-final",
```
Comment (kartang) [inline test/WebApiShimFw46.FunctionalTests/project.json]: I believe that this is their latest version.

**Comment by kartang** [comment]
Looks good to me!

File: test/WebApiShimFw46.FunctionalTests/project.json (hunk 2 above)
Comment (dnduffy) [inline test/WebApiShimFw46.FunctionalTests/project.json]: Yes, it is.

**Pull request merged**